static/css/documentation.css

```diff
@@ -898,17 +898,17 @@ a .mega-octicon {
 
 .dev-program-callout .wrapper {
   padding: 40px 0;
-  background: #160625 url(/images/callout-earth-static.png) no-repeat top right / 30%;
+  background: #FF9633 no-repeat top right / 30%;
```

Comment:
Can you remove 'no-repeat top right /30%' since the background is just a solid color now?